cluster/test-network.sh

```diff
@@ -24,7 +24,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-KUBE_CONFIG_FILE="config-default.sh"
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# export KUBE_CONFIG_FILE
+# use kubeconfig default file
+export KUBE_CONFIG_FILE="config-default.sh"
```

Comment:
Why did I do that: 
```
In ./cluster/test-network.sh line 28:
KUBE_CONFIG_FILE="config-default.sh"
^--------------^ SC2034: KUBE_CONFIG_FILE appears unused. Verify use (or export if used externally).
```

Reply:
I'm really wary of this one, what's using this downstream, ginkgo-e2e or something?

Reply:
If we do not select export, we can only delete.
https://github.com/koalaman/shellcheck/wiki/SC2034

Reply:
currently, ginkgo-e2e.sh seems to use KUBE_CONFIG_FILE="config-test.sh".
If it is exported in test-network.sh, ginkgo-e2e.sh uses config-default.sh instead of config-test.sh.
I think deleting is better for keeping current behavior...

Reply:
@spiffxp what do you think?

Reply:
Yeah, to maintain current behaviour, we should just remove `KUBE_CONFIG_FILE` from `cluster/test-network.sh`

Reply:
Agree, let's delete please